src/data/RESTSource.ts

```diff
@@ -0,0 +1,15 @@
+import semver from 'semver';
+
+import DocsSource from './DocsSource';
+
+const branchBlacklist = new Set(['docs']);
+export default new DocsSource({
+	id: 'rest',
+	name: 'REST',
+	global: 'REST',
+	docsRepo: 'discordjs/docs',
+	repo: 'discordjs/discord.js-modules',
+	defaultTag: 'stable',
+	branchFilter: (branch: string) => !branchBlacklist.has(branch) && !branch.startsWith('dependabot/'),
+	tagFilter: (tag: string) => semver.gt(tag.replace(/^v/, ''), '0.2.0'),
```

Comment:
This filter seems to throw because the tags over at [discord.js-modules](https://github.com/discordjs/discord.js-modules/tags) are not deemed semver-compatible in order to perform the comparison in the first place (`TypeError: Invalid Version: @discordjs/ws@0.1.1-canary.1` for example). Maybe attempt a try/catch here?

Reply:
Shouldn't be an issue, I'm not planning on releasing things like this anymore and old tags won't get a docs build 👍